proto/viam/common/v1/common.proto

```diff
@@ -89,8 +89,8 @@ message Mesh {
   bytes mesh = 2;
 }
 
-message Points {
-  repeated float array = 1;
+message PointCloud {
```

Comment:
I could go either way on this name but this does seem consistent with our nomenclature